custom/enikshay/tests/two_b_datamigration/test_logging.py

```diff
@@ -48,6 +48,7 @@ def drtb_import(self, import_rows, format, commit=False):
         get_users_path = \
             "custom.enikshay.two_b_datamigration.management.commands.import_drtb_cases.get_users_by_location_id"
         mock_user = MagicMock(
+            domain=self.domain,
```

Comment:
did you mean to add this in this PR?

Reply:
Yeah, this broke tests - `toggles.ENIKSHAY.enabled(user.domain)` fails if `user` is a `MagicMock`, since `user.domain` is expected to be a string.  That was not easy to track down :/